src/modules/ekf2/EKF/control.cpp

```diff
@@ -812,6 +812,8 @@ void Ekf::checkVerticalAccelerationHealth()
 		_clip_counter--;
 	}
 
+	_fault_status.flags.bad_acc_clipping = _clip_counter > clip_count_limit / 2;
```

Comment:
This changes the threshold from > 0 to >50 per EKF update. Do we have log data where this level of clipping was present without problems?

Reply:
50 is half where the ekf will start counting up for entry into a bad acceleration handling condition so looks like reasonable value for external reporting.

Reply:
This is what it looks like on a real vehicle before this EKF change.

![Screenshot from 2022-01-27 10-06-00](https://user-images.githubusercontent.com/84712/151385637-30a7dd0d-98be-4b94-b2cb-0f8b73746408.png)

![Screenshot from 2022-01-27 10-07-12](https://user-images.githubusercontent.com/84712/151385856-ccb00e4d-41cc-4859-a87c-95418793c51e.png)

It's these periods of fairly continuous clipping I'm trying to flag.

![Screenshot from 2022-01-27 10-08-11](https://user-images.githubusercontent.com/84712/151386157-5901fff3-f95b-412c-aa22-c29059c692c6.png)